server/src/routes/api/v1/movieReviewRouter.js

```diff
@@ -8,15 +8,17 @@ const movieReviewRouter = new express.Router({ mergeParams: true });
 
 movieReviewRouter.post("/", async (req, res) => {
   const movieId = req.params.id;
+  const userId = req.user.id;
   const { body } = req;
   const formInput = cleanUserInput(body);
-  const { content, userId } = formInput;
+  const { content, rating } = formInput;
 
   try {
     const newReview = await Review.query().insertAndFetch({
       movieId,
       userId,
       content,
+      rating,
     });
     return res.status(201).json({ newReview });
```

Comment:
should serialize the new review before sending to the front end so that it is inline with the rest of the serialized data